### DISCUSSION TITLE
Batch Not working is this just me?

### DISCUSSION THREAD

**Discussion by OscarAdams13** [Mikubill/sd-webui-controlnet#134] [General]
I am trying to run a batch png sequence  with controlnet and i am getting this error in webui:

AttributeError: 'numpy.ndarray' object has no attribute 'save'

has batch been working for other people?

**Comment by ljleb** [comment]
Check the previous discussion that occurred here: https://github.com/Mikubill/sd-webui-controlnet/discussions/111
> In settings tab, tick the "Do not append map to output" checkbox.

**Reply by ljleb** [reply]
Note that as of 4dab9f4790d7bae517ed3269fa98593f26708add this is not necessary anymore.

**Reply by OscarAdams13** [reply]
oh wow thanks !

**Reply by lyonnel4** [reply]
> Check the previous discussion that occurred here: #111
> 
> > In settings tab, tick the "Do not append map to output" checkbox.

Hi.
Since a recent update, I have no more ControlNet settings in the automatic1111 settings. So i can't enable the checkbox
is that normal ? (edit 2 : i saw the post about the bug. take your time to fix it. no need to answer this message and ty for your work.)
edit: my error is not the exactly same (it's mode and not save) : 
AttributeError: 'numpy.ndarray' object has no attribute 'mode'
